src/index.ts

```diff
@@ -464,14 +486,26 @@ export class Bigtable {
       {},
       baseOptions,
       {
-        servicePath: customEndpointBaseUrl || defaultAdminBaseUrl,
+        servicePath:
+          customEndpointBaseUrl ||
+          `bigtableadmin.${getDomain(options.BigtableTableAdminClient)}`,
```

Comment:
If no universe domain is provided then this will still be bigtable.googleapis.com.